frameworks/PHP/hyperf/benchmark_config.json

```diff
@@ -24,6 +24,52 @@
                 "display_name": "Hyperf",
                 "notes": "",
                 "versus": "swoole"
+            },
+            "raw": {
+                "json_url": "/json",
```

Comment:
Hi @huangzhhui 
You don't need to add all these urls that are the same as the basic **hyperf** version:
raw-urls are sufficient:  `json`, `plaintext` and `fortune` are useless  and overload the results for nothing.